### PR TITLE
fix grammatical issues in working with JSON

### PR DESCRIPTION
**Pull request by yashikabhargava**
### Description

fixed minor grammatical issues in working with JSON.

### CODE REVIEW

File: files/en-us/learn/javascript/objects/json/index.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ This allows you to construct a data hierarchy, like so:
 }
 ```
 
-If we loaded this string into a JavaScript program, parsed it into a variable called `superHeroes` for example, we could then access the data inside it using the same dot/bracket notation we looked at in the [JavaScript object basics](/en-US/docs/Learn/JavaScript/Objects/Basics) article.
+If we loaded this string into a JavaScript program, and parsed it into a variable called `superHeroes`. For example, we could then access the data inside it using the same dot/bracket notation we looked at in the [JavaScript object basics](/en-US/docs/Learn/JavaScript/Objects/Basics) article.
```
Comment (Josh-Cena) [inline files/en-us/learn/javascript/objects/json/index.md:101]: The "for example" is modifying "a variable called `superHeroes`". I don't have good suggestions for how to make it sound better.

Comment (yashikabhargava) [reply files/en-us/learn/javascript/objects/json/index.md:101]: Yes! I'm sorry, I misunderstood that will undo the change.

Comment (yashikabhargava) [reply files/en-us/learn/javascript/objects/json/index.md]: I thought adding a full stop after "for example" might make it sound more clear, can you please review if it's correct or not, and if it doesn't work then let me know I'll replace it with the original one.

File: files/en-us/learn/javascript/objects/json/index.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ This allows you to construct a data hierarchy, like so:
 }
 ```
 
-If we loaded this string into a JavaScript program, parsed it into a variable called `superHeroes` for example, we could then access the data inside it using the same dot/bracket notation we looked at in the [JavaScript object basics](/en-US/docs/Learn/JavaScript/Objects/Basics) article.
+If we loaded this string into a JavaScript program, and parsed it into a variable called `superHeroes` for example. We could then access the data inside it using the same dot/bracket notation we looked at in the [JavaScript object basics](/en-US/docs/Learn/JavaScript/Objects/Basics) article.
```
Comment (Josh-Cena) [inline files/en-us/learn/javascript/objects/json/index.md]:
```suggestion
If we loaded this string into a JavaScript program and parsed it into a variable called `superHeroes` for example, we could then access the data inside it using the same dot/bracket notation we looked at in the [JavaScript object basics](/en-US/docs/Learn/JavaScript/Objects/Basics) article.
```

**Review by Josh-Cena — APPROVED** [review]
Thanks

**Pull request merged**